relay-config/src/config.rs

```diff
@@ -1451,6 +1454,11 @@ impl Config {
         self.values.limits.max_attachments_size.as_bytes()
     }
 
+    /// Returns the maxmium combined size of client reports in bytes.
```

Comment:
Not sure if we want this config flag, but if we do, we probably need to document it.